### PR TITLE
libid3tag: update to 0.16.3

### PR DESCRIPTION
**Pull request by krant**
Maintainer: @thess 
Run tested: ARMv7, Linksys WRT3200ACM, master branch

Description:
- Update package URLs to official ones
- Switch to CMake build
- Drop custom .pc file in favor of upstream version

### CODE REVIEW

File: libs/libid3tag/Makefile [hunk 1]
```diff
@@ -8,61 +8,44 @@
...
 PKG_NAME:=libid3tag
-PKG_VERSION:=0.15.1b
-PKG_RELEASE:=4
+PKG_VERSION:=0.16.3
+PKG_RELEASE:=1
 
-PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
-PKG_SOURCE_URL:=@SF/mad
-PKG_HASH:=63da4f6e7997278f8a3fef4c6a372d342f705051d1eeb6a46a86b03610e26151
+PKG_SOURCE:=$(PKG_VERSION).tar.gz
```
Comment (BKPepe) [inline libs/libid3tag/Makefile:14]: Nope.

**Review by BKPepe — CHANGES_REQUESTED** [review]

File: libs/libid3tag/Makefile (hunk 1 above)
Comment (neheb) [reply libs/libid3tag/Makefile:14]: What he said. If upstream doesn't create proper tarballs, switch to OpenWrt ones with PKG_SOURCE_PROTO:=git

Such a thing needs a treewide commit IMO.

Comment (krant) [reply libs/libid3tag/Makefile]: Switched to git source

File: libs/libid3tag/Makefile [hunk 2]
```diff
@@ -8,61 +8,46 @@
...
 PKG_NAME:=libid3tag
-PKG_VERSION:=0.15.1b
-PKG_RELEASE:=4
+PKG_VERSION:=0.16.3
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://codeberg.org/tenacityteam/libid3tag.git
+PKG_SOURCE_DATE:=2023-09-24
+PKG_SOURCE_VERSION:=e02ecf1276b467a8a5dd20c55ce711e6f7116d3e
```
[the remark this answers is not in the document]
Comment (krant) [reply libs/libid3tag/Makefile]: Oops, now there is, thanks :slightly_smiling_face:

File: libs/libid3tag/Makefile [hunk 3]
```diff
@@ -8,61 +8,46 @@
...
 
 PKG_NAME:=libid3tag
-PKG_VERSION:=0.15.1b
-PKG_RELEASE:=4
+PKG_VERSION:=0.16.3
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://codeberg.org/tenacityteam/libid3tag.git
+PKG_SOURCE_DATE:=2023-09-24
```
Comment (neheb) [inline libs/libid3tag/Makefile:16]: With that git tag, no need for a date.

**Comment by diizzyy** [comment]
..or just use the same trick as with GitHub
https://cgit.freebsd.org/ports/tree/audio/libid3tag/Makefile#n4

**Pull request merged**